daemon/main.c

```diff
@@ -321,7 +321,7 @@ int help(int exitcode) {
             " License    : https://github.com/netdata/netdata/blob/master/LICENSE.md\n"
             "\n"
             " Twitter    : https://twitter.com/linuxnetdata\n"
-            " Facebook   : https://www.facebook.com/linuxnetdata/\n"
+            " LinkedIn   : https://linkedin.com/company/netdata-cloud/\n"
```

Comment:
Hello @andrewm4894 ,

Why cannot we have Facebook and LinkedIn? :thinking: 

Best regards!

Reply:
No reason, just if had to pick one I'd imagine LinkedIn be better. I didn't want to add a new line or change it that much in case had any other knock on impacts.

Reply:
I think just adding LinkedIn is ok, agree with @thiagoftsm 

regardless of changes, we need to make them in here too

https://github.com/netdata/netdata/blob/a364cef58290bb8b699f2bf356bf086663d67002/daemon/README.md?plain=1#L113-L130

Reply:
Probably we should give for users everywhere they can reach us instead hide options @andrewm4894, this will simplify our users life. :)

Reply:
We can add @thiagoftsm email there too

Reply:
done